lnrpc/lightning.proto

```diff
@@ -2847,6 +2847,11 @@ message PendingChannelsResponse {
         // very likely canceled the funding and the channel will never become
         // fully operational.
         int32 funding_expiry_blocks = 3;
+
+        // The number of blocks remaining until the channel status changes from
+        // pending to active. A value of 0 indicates that the channel is now
```

Comment:
Also include following comments:

1. Active here means that both channel peers consider this channel as open and can start using it. For public channels this does not mean that the channel is broadcasted to the network yet, this only happens after 6 confirmations which is a BOLT specification. (Link the specifc line of the spec here)

2. ZeroConf channels are active from the beginning which means they never show up as pending from the start.